resources/assets/coffee/react/beatmaps/main.coffee

```
@@ -250,12 +259,20 @@ export class Main extends React.PureComponent
     @setState loading: true
     @backToTop.current.reset()
 
-    @fetchNewState(true).then (newState) =>
+    @fetchNewState(true)
+    .then (newState) =>
       cutoff = @backToTopAnchor.current.getBoundingClientRect().top
       window.scrollTo window.pageXOffset, window.pageYOffset + cutoff if cutoff < 0
 
       @setState newState
 
+    .catch (error) =>
+      osu.ajaxError error
```

it needs to ignore error from `.abort()` call.